src/Pages/MyBag/MyBag.js

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router';
+
+import SubEvent from './SubEvent/SubEvent';
+import OrderProduct from '../../Components/OrderProduct/OrderProduct';
+import OrderSummary from './OrderSummary/OrderSummary';
+import Button from '../../Components/Button/Button';
+
+import './MyBag.scss';
+
+const MyBag = () => {
+  const navigate = useNavigate();
+  const [orderProducts, setOrderProducts] = useState([]);
+
+  const getData = async () => {
+    const data = await (await fetch('/data/orderProducts.json')).json();
+
```

Comment:
await가 중복되어 있어 가독성이 좋지 않을 것 같습니다.
```suggestion
  const getData = async () => {
    const data = await fetch('/data/orderProducts.json').then(res => res.json());

```

Reply:
네 수정하였습니다!